### PR TITLE
fix #19 Add Circle ci

### LINKED ISSUE #19: add CI
# Issue
We should have CI. I hear circleCI is better than Travis.

# Proposed Implementation
Add beacon_chain repo to circleCI, configured to run pytest tests

### PR DESCRIPTION
**Pull request by ChihChengLiang**
This might need some project configuration:

1. setting up `etherum/beacon_chain` project on https://circleci.com . This should also add the pull requests webhook to Github.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6.1
```
Comment (jannikluhn) [inline .circleci/config.yml]: Any reason not to use the most up to date version here (3.6.5)? [py-EVM](https://github.com/ethereum/py-evm/blob/master/.circleci/config.yml#L52) specifies just `circleci/python:3.6` which presumably chooses the most recent one available.

Comment (ChihChengLiang) [reply .circleci/config.yml]: Ah ... because that comes from the template that circle ci provides. `circleci/python:3.6` sounds good, will fix to this

**Comment by djrtwo** [comment]
@ChihChengLiang Were you able to setup the project on circleci?

**Comment by ChihChengLiang** [comment]
Not yet, I've sent a request to devops team for the permission to do the config

**Comment by ChihChengLiang** [comment]
Ok, CI configured. Once this merged, we'll have the ci.

**Comment by djrtwo** [comment]
Great! Going to merge and port into my branch to test.

**Pull request merged**